agent/exploits/cve_2024_11639.py

```diff
@@ -61,12 +58,12 @@ def check(self, target: definitions.Target) -> list[definitions.Vulnerability]:
             logging.error("HTTP Request failed: %s", e)
             return []
 
-        if HEADER_LAST_MODIFIED not in resp.headers:
+        if HEADER_IF_MODIFIED_SINCE not in resp.headers:
```

Comment:
This name is kind of confusion, it feels like some kind of boolean, but it's a string
Does it get None in some cases or smething?

Reply:
OK